backends-velox/src/test/scala/org/apache/gluten/execution/TableFunctionsValidateSuite.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gluten.execution
+
+class TableFunctionsValidateSuite extends FunctionsValidateTest {
+  disableFallbackCheck
+
+  test("Test json_tuple function") {
```

Comment:
This test looks good. However, the tests of offloading expressions are places in `TestOperator`. Is there any special reason to not put this test there?

Reply:
moved to `TestOperator`